django_socio_grpc/servicer_proxy.py

```diff
@@ -73,6 +76,7 @@ def handler(request, context):
                 finally:
                     # INFO - AM - 22/04/2021 - next line break tests. Need to more understand the drowback about memory in production
                     # db.close_old_connections()
+                    close_old_connections()
                     pass
```

Comment:
remove the pass and the comments aboves.
First comment change it to:
`# INFO - AM - 30/06/2021 - Need this in production environnement to avoid SSL end of files errors when too much connection on database`